pydy/system.py

```diff
@@ -610,3 +610,42 @@ def _Kane_constant_symbols(self):
                 *self._Kane_inlist_insyms()))
         constants.remove(dynamicsymbols._t)
         return constants
+
+    def add_gravity(self, direction=None):
```

Comment:
Instead of add_gravity, it should be more generalized to *add and apply* different forces, and torques(if possible) in a single generic ClassMethod